src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -183,7 +183,7 @@ void addONNXToZHighPasses(mlir::PassManager &pm) {
   // sub, ...) that are of `stick -> light-weight op -> unstick`, it's better to
   // use CPU instead of NNPA to avoid stick/unstick. CPU is efficient to handle
   // these ops, e.g vectorize the computation.
-  if (nnpaEnableZHighToOnnx)
+  if (! nnpaDisableZHighToOnnx)
```

Comment:
Looks like the space caused `clang-format check` failed.

Reply:
Tx